testutils/testutils-kmp/src/commonMain/kotlin/androidx/kruth/MapSubject.kt

```diff
@@ -35,4 +35,17 @@ class MapSubject<K, V> internal constructor(actual: Map<K, V>?) : Subject<Map<K,
             asserter.fail("Expected to contain $key, but was ${actual.keys}")
         }
     }
+
+    /** Fails if the map does not contain exactly the given set of entries in the given map. */
```

Comment:
```suggestion
    /**
     * Fails if the map does not contain exactly the keys in the given map, mapping to values that
     * correspond to the values of the given map.
     */
```

Can we maintain the existing docs?

Reply:
I think that's the docs for the other `containsExactlyEntriesIn`. I took these docs from [this function definition](https://github.com/google/truth/blob/7d4a851d1b44ce054c0984ae96c97fb6cfcbd1f0/core/src/main/java/com/google/common/truth/MapSubject.java#L227-L244).

Reply:
Ah yes you're right - my mistake!